thorlcr/activities/thdiskbase.cpp

```diff
@@ -48,6 +48,10 @@ void CDiskReadMasterBase::init()
     Owned<IDistributedFile> file = queryThorFileManager().lookup(container.queryJob(), helperFileName, 0 != ((TDXtemporary|TDXjobtemp) & helper->getFlags()), 0 != (TDRoptional & helper->getFlags()), true);
     if (file)
     {
+        CDfsLogicalFileName lfn;
```

Comment:
Would it be worth adding a file->isExternal() call?  It would save processing the filename again.